packages/frontend/src/components/ChartConfigPanel/ChartConfigTabs.tsx

```diff
@@ -78,7 +78,7 @@ const ChartConfigTabs: FC = () => {
                 }),
                 {},
             );
-    }, [dirtyEchartsConfig]);
+    }, [dirtyEchartsConfig, orgData]);
```

Comment:
is this orgData needed here ?

Reply:
Yes! Inside the reduce there's call to `orgData` -> `orgData?.chartColors ...` L76